README.md

```diff
@@ -14,19 +14,8 @@ ODS Pipeline is well suited for regulated development (e.g. medical device softw
 
 ## Documentation
 
-The documentation provided by ODS pipeline has three audiences:
-
-* **Admins** install and maintain a central ODS pipeline installation in an OpenShift cluster that can be used by many users.
-
-* **Users** consume an existing central ODS pipeline installation. Users install ODS pipeline resources into a namespace they own to run CI/CD pipelines for their repositories.
-
-* **Contributors** work on the ODS pipeline project itself, for example by improving existing tasks, adding new ones, updating documentation, etc.
-
-**Admin Guide**
-* [Installation & Updating](/docs/admin-installation.adoc)
-
 **User Guide**
```

Comment:
I would like to remove `User Guide` completely. I'd move the `Contributor Guide` below to a separate section in the readme called `## Contributing` (I believe this should be the last section in the readme)